core/src/main/java/com/novoda/merlin/receiver/ConnectivityChangeEventCreator.java

```diff
@@ -8,13 +8,13 @@
 class ConnectivityChangeEventCreator {
 
     ConnectivityChangeEvent createFrom(Intent intent, MerlinsBeard merlinsBeard) {
-        boolean isConnected = getIsConnectedFrom(intent, merlinsBeard);
+        boolean isConnected = extractIsConnectedFrom(intent, merlinsBeard);
         String info = intent.getStringExtra(ConnectivityManager.EXTRA_EXTRA_INFO);
         String reason = intent.getStringExtra(ConnectivityManager.EXTRA_REASON);
         return ConnectivityChangeEvent.createWithNetworkInfoChangeEvent(isConnected, info, reason);
     }
 
-    private boolean getIsConnectedFrom(Intent intent, MerlinsBeard merlinsBeard) {
+    private boolean extractIsConnectedFrom(Intent intent, MerlinsBeard merlinsBeard) {
```

Comment:
`ConnectionStatus` sounds like a different object, when we are only interested in a boolean 🤔